include/deal.II/cgal/triangulation.h

```diff
@@ -18,17 +18,32 @@
 
 #include <deal.II/base/config.h>
 
-#ifdef DEAL_II_WITH_CGAL
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/function.h>
+
+#include <deal.II/grid/tria.h>
 
-#  include <deal.II/grid/tria.h>
+#include <deal.II/cgal/utilities.h>
+
+#ifdef DEAL_II_WITH_CGAL
 
 #  include <boost/hana.hpp>
 
+#  include <CGAL/Complex_2_in_triangulation_3.h>
+#  include <CGAL/IO/facets_in_complex_2_to_triangle_mesh.h>
+#  include <CGAL/Implicit_surface_3.h>
+#  include <CGAL/Labeled_mesh_domain_3.h>
+#  include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#  include <CGAL/Mesh_criteria_3.h>
+#  include <CGAL/Mesh_triangulation_3.h>
 #  include <CGAL/Polyhedron_3.h>
 #  include <CGAL/Surface_mesh.h>
+#  include <CGAL/Surface_mesh_default_triangulation_3.h>
 #  include <CGAL/Triangulation_2.h>
 #  include <CGAL/Triangulation_3.h>
-#  include <deal.II/cgal/utilities.h>
+#  include <CGAL/make_mesh_3.h>
+#  include <CGAL/make_surface_mesh.h>
+#  include <deal.II/cgal/surface_mesh.h>
```

Comment:
Why do you suddenly need to include more header files?

Reply:
I moved them from `grid_generator.h`. This should be fixed as soon as I am able to address the comments above.